src/stactools/core/addraster.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from osgeo import gdal
```

Comment:
I don't believe `gdal` is an explicit dependency in https://github.com/stac-utils/stactools/blob/7aa3ae09e04aa7ebb48ac24da649979d32367f34/setup.cfg#L33-L42, so it should be added there.

Reply:
does it have to be explicit? it is a requirement for rasterio, so if rasterio is installed, gdal should be as well.

Reply:
It does. GDAL's system libraries are a requirement for rasterio, but not its Python packages. E.g. this is from my personal stactools virtualenv:

```
$ python -c 'import stactools.core; print(stactools.core.__version__)'
0.2.2
$ python -c 'from osgeo import gdal'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'osgeo'
```

Reply:
Right, good catch. I will add it